apollo-backend/src/main/java/io/logz/apollo/blockers/SingleRegionBlockerResponse.java

```diff
@@ -0,0 +1,38 @@
+package io.logz.apollo.blockers;
+
+import java.util.List;
+
+public class SingleRegionBlockerResponse {
+    public static final String BLOCKER_NAME = "ServiceByRegionBlocker";
```

Comment:
again, this is a general response for potentially multiple RequestBlockers
you need to pass either **blockerType** or **blockerName** as parameter via constructor and delete this constant

Reply:
Done.